dawn/main/ansible/Ansible.js

```diff
@@ -212,15 +216,15 @@ class TCPSocket {
 
   waitRuntimeConfirm(message, count) {
     if (count > 3) {
-      console.error('Runtime failed to confirm');
+      this.logger.error('Runtime failed to confirm');
```

Comment:
This doesn't appear like it'll work, as the `logger` class has no `error` method.